### PR TITLE
Added RDS failover tips and gotchas

### PR DESCRIPTION
**Pull request by jlordiales**

### CODE REVIEW

**Comment by forwidur** [comment]
LGTM. There are conflicts though, would you mind taking a look?

Thanks a lot, @jlordiales !

**Comment by jlordiales** [comment]
Should be ok now, thanks!

File: README.md [hunk 1]
```diff
@@ -981,9 +982,9 @@ RDS
 
 -	⏱RDS instances run on EBS volumes, and hence are constrained by the EBS performance.
 -	🔸Verify what database features you need, as not everything you might want is available on RDS. For example, if you are using Postgres, check the list of [supported features and extensions](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#SQLServer.Concepts.General.FeatureSupport). If the features you need aren’t supported by RDS, you’ll have to deploy your database yourself.
+- If you use the failover support offered by RDS keep in mind that it is based on DNS changes. Therefore, you need to make sure that your client reacts to these changes appropriately. This is particularly important for Java, given how the [JVM TTL](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-jvm-ttl.html) is configured by default.
```
Comment (ThanosBaskous) [inline README.md]: nit: missing comma
"If you use the failover support offered by RDS keep in mind..." -> "If you use the failover support offered by RDS, keep in mind..."

Comment (ThanosBaskous) [inline README.md]: brevity suggestion: "...based on DNS changes. Therefore, you need to make sure..." -> "...based on DNS changes – make sure..."

Comment (ThanosBaskous) [inline README.md]: phrasing suggestion: "This is particularly important in Java, given how its DNS resolver's TTL is [configured by default](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-jvm-ttl.html)."

**Comment by jlordiales** [comment]
Thanks for the review @ThanosBaskous! I added your suggestions.

File: README.md [hunk 2]
```diff
@@ -989,9 +990,9 @@ RDS
 
 -	⏱RDS instances run on EBS volumes, and hence are constrained by the EBS performance.
 -	🔸Verify what database features you need, as not everything you might want is available on RDS. For example, if you are using Postgres, check the list of [supported features and extensions](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#SQLServer.Concepts.General.FeatureSupport). If the features you need aren’t supported by RDS, you’ll have to deploy your database yourself.
+- If you use the failover support offered by RDS, keep in mind that it is based on DNS changes - make sure that your client reacts to these changes appropriately. This is particularly important for Java, given how its DNS resolver's TTL is [configured by default](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-jvm-ttl.html).
```
Comment (ThanosBaskous) [inline README.md]: Nit: convention in the guide is to use unicode apostrophes. "resolver's" -> "resolver‘s"

Comment (jlordiales) [inline README.md]: Fixed!

File: README.md [hunk 3]
```diff
@@ -989,9 +990,9 @@ RDS
 
 -	⏱RDS instances run on EBS volumes, and hence are constrained by the EBS performance.
 -	🔸Verify what database features you need, as not everything you might want is available on RDS. For example, if you are using Postgres, check the list of [supported features and extensions](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#SQLServer.Concepts.General.FeatureSupport). If the features you need aren’t supported by RDS, you’ll have to deploy your database yourself.
+- If you use the failover support offered by RDS, keep in mind that it is based on DNS changes - make sure that your client reacts to these changes appropriately. This is particularly important for Java, given how its DNS resolver’s TTL is [configured by default](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-jvm-ttl.html).
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit (commenting for future reference) - please use em dashes (–) instead of hyphens (-).

**Pull request merged**